crates/compiler/solve/tests/solve_expr.rs

```diff
@@ -522,6 +522,41 @@ mod solve_expr {
         );
     }
 
+    #[test]
+    fn my_bug() {
```

Comment:
Can this test be given a more descriptive name, e.g. "choose_correct_recursion_var_under_record"?